lib/algos/list/delimiterCurrent.ts

```diff
@@ -43,9 +49,57 @@ class DelimiterCurrent extends Delimiter {
                 ne: this.excludedDataStoreName,
             }
         }
+
+        this.start = Date.now();
+
         return params;
     }
 
+    _parse(s) {
```

Comment:
A little JSDoc could help, if only to mention that it may return an object or `undefined` if parsing fails